src/protocols/bitcoin/views/index.ts

```diff
@@ -9,8 +9,10 @@ const protocolViews: ProtocolViewsConfig = {
     /* webpackChunkName: "btc-account-details-transactions" */
     './AccountDetailsTransactions.vue'
   ),
+  AccountDetailsTokens: null,
+  AccountDetailsNames: null,
```

Comment:
So from now on on adding new blockchains we would need to set `AccountDetailsTokens` and `AccountDetailsNames` to null if it is not supported?